docs/acme-integration.rst

```diff
@@ -156,28 +156,44 @@ configuration active, it will check for validity of the certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
-Certificate for subdomains excluding the apex domain
-----------------------------------------------------
+Example: Certificate for apex domain and subdomains
```

Comment:
IMO "domain" is wrong. certificates are issued for *hosts*, not for domains. For domains one would need to issue 2 `*`-certificate.